src/js/common/schemaform/definitions/address.js

```diff
@@ -39,7 +39,8 @@ export function schema(currentSchema, isRequired = false) {
       },
       state: {
         title: 'State',
-        type: 'string'
+        type: 'string',
+        maxLength: 51
```

Comment:
I noticed this when looking at the address schema, so adding it just to be careful.